### DISCUSSION TITLE
Add a way to disable Minecraft's default anti noclip in config + permission

### DISCUSSION THREAD

**Discussion by yorik100** [PurpurMC/Purpur#1334] [Feature Requests]
It would be very useful for Bedrock edition users to avoid them glitching when going upstairs, I handle these noclip checks with another plugin.

**Comment by 1ByteBit** [comment]
> It would be very useful for Bedrock edition users

Purpur most likely isn't going to add fixes for bedrock edition users, as this is a Java edition software.

See if the geyser devs are able to do anything about it, as they're the ones allowing people to do incompatible things.

**Reply by yorik100** [reply]
> > It would be very useful for Bedrock edition users
> 
> Purpur most likely isn't going to add fixes for bedrock edition users, as this is a Java edition software.
> 
> See if the geyser devs are able to do anything about it, as they're the ones allowing people to do incompatible things.

You cannot disable it, you just cannot disable it at all without modifying the server jar, it literally is impossible, only solution would be to block teleport event entirely

**Reply by 1ByteBit** [reply]
> You cannot disable it, you just cannot disable it at all without modifying the server jar, it literally is impossible, only solution would be to block teleport event entirely

No, you can't disable it, but you can still workaround it. And as I implied a moment ago, this is a fix for something which is incompatible by nature. You're going to have to seek support from the people who allow you to do such things in the first place. I don't want to speak for the devs, but it has been made clear before where they stand on protocolhacks within their discord, I highly doubt they're going to support such a thing by adding a fix which is only useful to this
